lib/sapine.rb

```diff
@@ -78,6 +79,10 @@ def per_page
     params[:per_page] ? params[:per_page].to_i : DEFAULT_PER_PAGE
   end
 
+  def page
+    params[:page] ? params[:page].to_i : DEFAULT_PAGE
```

Comment:
ActiveRecord should just return empty results. If that's ok with you guys, you can leave as is.